### PR TITLE
Add echo endpoint handler.

### PR DESCRIPTION
**Pull request by taik0**

### CODE REVIEW

File: config/config_test.go [hunk 1]
```diff
@@ -210,7 +210,7 @@ func TestConfig_init(t *testing.T) {
 		t.Error(err.Error())
 	}
 
-	if hash != "ip9OZeBfyY9vrjovdU+IivbOev4bH1jDcT2EC9QU1U8=" {
+	if hash != "N1AP3YSocze/xZpcFmMflCIaqvBNyifukZXKFTj856Y=" {
```
Comment (dhontecillas) [inline config/config_test.go:213]: 🤔 I am not sure what are we testing here ... do we need this test ?

File: router/gin/echo_test.go [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+	"testing"
+
+	"github.com/gin-gonic/gin"
+)
+
+func TestEchoHandler(t *testing.T) {
+	reqBody := `{"message":"some body to send"}`
+	expectedRespBody := `{"req_uri":"http://127.0.0.1:8088/_gin_endpoint/a?b=1","req_uri_details":{"fragment":"","host":"127.0.0.1:8088","path":"/_gin_endpoint/a","query":"b=1","scheme":"http","user":""},"req_method":"GET","req_querystring":{"b":["1"]},"req_body":"{\"message\":\"some body to send\"}","req_headers":{"Content-Type":["application/json"]}}`
+	expectedRespNoBody := `{"req_uri":"http://127.0.0.1:8088/_gin_endpoint/a?b=1","req_uri_details":{"fragment":"","host":"127.0.0.1:8088","path":"/_gin_endpoint/a","query":"b=1","scheme":"http","user":""},"req_method":"GET","req_querystring":{"b":["1"]},"req_body":"","req_headers":{"Content-Type":["application/json"]}}`
+	expectedRespString := `{"req_uri":"http://127.0.0.1:8088/_gin_endpoint/a?b=1","req_uri_details":{"fragment":"","host":"127.0.0.1:8088","path":"/_gin_endpoint/a","query":"b=1","scheme":"http","user":""},"req_method":"GET","req_querystring":{"b":["1"]},"req_body":"Hello lura","req_headers":{"Content-Type":["application/json"]}}`
```
Comment (dhontecillas) [inline router/gin/echo_test.go:19]: I would prefer to have the string with carry returns for easier readability, but, this is fine.

File: router/mux/echo_test.go [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+	"net/http/httptest"
+	"strings"
+	"testing"
+)
+
+func TestEchoHandlerNew(t *testing.T) {
+	reqBody := `{"message":"some body to send"}`
+	expectedRespBody := `{"req_uri":"http://127.0.0.1:8088/_gin_endpoint/a?b=1","req_uri_details":{"fragment":"","host":"127.0.0.1:8088","path":"/_gin_endpoint/a","query":"b=1","scheme":"http","user":""},"req_method":"GET","req_querystring":{"b":["1"]},"req_body":"{\"message\":\"some body to send\"}","req_headers":{"Content-Type":["application/json"]}}`
+	expectedRespNoBody := `{"req_uri":"http://127.0.0.1:8088/_gin_endpoint/a?b=1","req_uri_details":{"fragment":"","host":"127.0.0.1:8088","path":"/_gin_endpoint/a","query":"b=1","scheme":"http","user":""},"req_method":"GET","req_querystring":{"b":["1"]},"req_body":"","req_headers":{"Content-Type":["application/json"]}}`
+	expectedRespString := `{"req_uri":"http://127.0.0.1:8088/_gin_endpoint/a?b=1","req_uri_details":{"fragment":"","host":"127.0.0.1:8088","path":"/_gin_endpoint/a","query":"b=1","scheme":"http","user":""},"req_method":"GET","req_querystring":{"b":["1"]},"req_body":"Hello lura","req_headers":{"Content-Type":["application/json"]}}`
```
Comment (dhontecillas) [inline router/mux/echo_test.go:17]: Here I would also add carry returns and indentation to the strings for better readability, but again .. is just a personal preference.

**Review by dhontecillas — APPROVED** [review]
Looks good to me: just small comments about personal opinions ;)

**Review by kpacha — APPROVED** [review]

**Pull request merged**